docs/overview/Lifecycle.md

```diff
@@ -0,0 +1,157 @@
+<!--ONE-DOCS-HIDE start-->
+[👈 Return to Overview](./README.md)
+<!--ONE-DOCS-HIDE end-->
+
+# One App Life Cycles
+
+In this guide, we'll break down what happens when One App starts up,
+renders a Holocron module and how you can configure the behavior of
+One App (and it's life cycles) via configuration.
+
+## 📖 Table of Contents
+* [One App Server](#one-app-server)
+  * [Holocron Runtime](#holocron-runtime)
+  * [Advanced Settings](#advanced-settings)
+* [Rendering](#rendering)
+
+## One App Server
+
+When the One App `node.js` server first boots up, there are a series of steps
+undertaken. The first step One App takes is configuration, which can be done via
+command line arguments and environment variables.
+
+**Configuration**
+
+Whenever One App is ran `NODE_ENV` must be defined with either
+`development` or `production`. One App behaves differently depending
+on which node environment it is running under. In development mode,
+One App spins up a development CDN and API server that run on
```

Comment:
We should link to https://github.com/americanexpress/one-app-dev-cdn here

Reply:
Updated with links (10a7806)